docs/reference/storage-uri.md

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
The "Storage URI" page feels weird in the reference section. Will fix in a follow-up PR.